src/crypto/BUILD.gn

```diff
@@ -60,7 +60,11 @@ static_library("crypto") {
   public_configs = []
   if (chip_crypto == "mbedtls") {
     sources += [ "CHIPCryptoPALmbedTLS.cpp" ]
-    public_deps += [ "${mbedtls_root}:mbedtls" ]
+
+    # ESP32 has its own mbedTLS
+    if (target_cpu != "esp32") {
+      public_deps += [ "${mbedtls_root}:mbedtls" ]
```

Comment:
Remove this and set the |mbedtls_target| argument.

Reply:
fixed